frontend/src/routes/users/me/+page.svelte

```diff
@@ -0,0 +1,170 @@
+          <div class="flex-shrink-0">
+            <!-- Heroicon name: mini/check-circle -->
+            <Icon name="CheckCircle" />
+          </div>
+          <div class="ml-3">
+            <p class="text-sm font-medium text-green-800">Successfully uploaded</p>
+          </div>
+          <div class="ml-auto pl-3">
+            <div class="-mx-1.5 -my-1.5">
+              <button
+                type="button"
+                class="inline-flex rounded-md bg-green-50 p-1.5 text-green-500 hover:bg-green-100 focus:outline-none focus:ring-2 focus:ring-green-600 focus:ring-offset-2 focus:ring-offset-green-50"
+                on:click|preventDefault={closeAlert}
+              >
+                <span class="sr-only">Dismiss</span>
+                <!-- Heroicon name: mini/x-mark -->
+                <svg
+                  class="h-5 w-5"
+                  xmlns="http://www.w3.org/2000/svg"
+                  viewBox="0 0 20 20"
+                  fill="currentColor"
+                  aria-hidden="true"
+                >
+                  <path
+                    d="M6.28 5.22a.75.75 0 00-1.06 1.06L8.94 10l-3.72 3.72a.75.75 0 101.06 1.06L10 11.06l3.72 3.72a.75.75 0 101.06-1.06L11.06 10l3.72-3.72a.75.75 0 00-1.06-1.06L10 8.94 6.28 5.22z"
+                  />
+                </svg>
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+    {/if}
+    <form action="#" method="POST">
+      <div class="shadow sm:overflow-hidden sm:rounded-md">
+        <div class="space-y-6 bg-white py-6 px-4 sm:p-6">
+          <div>
+            <h3 class="text-lg font-medium leading-6 text-gray-900">Profile</h3>
+            <p class="mt-1 text-sm text-gray-500">
+              This information will be displayed publicly so be careful what you share.
+            </p>
+          </div>
+        </div>
+
+        <!-- Datapicker -->
+
+        <div class="space-y-6 sm:px-6 lg:col-span-9 lg:px-0">
+          <div class="shadow sm:overflow-hidden sm:rounded-md">
+            <div class="space-y-6 bg-white py-6 px-4 sm:p-6">
+              <div>
+                <h3 class="text-lg font-medium leading-6 text-gray-900">Input your birthdate</h3>
+                <form>
+                  <div class="date-picker">
+                    <input
+                      bind:value={birthdate}
+                      class="mt-1 rounded-md border-gray-300 shadow-sm focus:border-indigo-500 focus:ring-indigo-500 sm:text-sm"
+                      type="date"
+                      id="party"
+                      name="party"
+                      min="1900-01-01"
+                      max="2099-01-01"
+                      required
+                    />
+                    <div class="inline-block">
+                      <button
+                        type="submit"
+                        class="mt-1 inline-block rounded-lg bg-green-500 px-5 py-3 text-sm font-medium text-white invalid:border-red-500"
```

Comment:
Green button looks kinda out-of-design here.
Let's use buttons as defined in TailwindUI:
https://tailwindui.com/components/application-ui/elements/buttons